### PR TITLE
Created Boulder file

### LINKED ISSUE #22: @codeangler  - tracking workflow
# Workflow
- [x] Create a branch
- [x] Make commits on new branch
- [x] Open a pull request
- [ ] Collaborate 🎉 
- [ ] Make more commits
- [ ] Merge into master

### PR DESCRIPTION
**Pull request by codeangler**
This pull request closes #22 

This pull request:
- Creates file for Boulder
- Adds activities for Boulder, CO

@beardofedu  @brianamarie   What are you favorite :beers:  in Boulder, CO? Have you been to the github office on Walnut in Boulder, CO?  Is there a way to visit :grey_question:

### CODE REVIEW

File: codeangler-boulder.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Boulder
+
+### Things to do
+- Fly-fishing
```
Comment (seanmishra) [inline codeangler-boulder.md:4]: No idea what that is, but would love to try. :+1:

File: codeangler-boulder.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Upslope Brewery
+- free climb 3rd flatiron :hamburger
```
Comment (pjlarson) [inline codeangler-boulder.md:6]: beer and outdoors, sounds like fun :confetti_ball: :evergreen_tree:

**Comment by seanmishra** [comment]
Boulder, CO deserves a merge.

File: codeangler-boulder.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Boulder
```
Comment (pjlarson) [inline codeangler-boulder.md:1]: Can you breathe OK at a mile high?

Comment (codeangler) [inline codeangler-boulder.md:1]: @pjlarson easily.  I'm built like a sherpa: all torso.

File: codeangler-boulder.md (hunk 1 above)
Comment (codeangler) [inline codeangler-boulder.md:4]: If you ever want to visit. I guide trips part-time for beginners.

**Comment by chuntr** [comment]
:fishing_pole_and_fish: :+1:

**Comment by codeangler** [comment]
@chuntr  thanks

**Comment by brianamarie** [comment]
Looks good! 👍 I've never been to the Boulder office, and I'm not sure if they have visitors...but I know the SF office allows them on the 1st floor!

**Pull request merged**